### ISSUE TITLE
[Accessibility] [HC] ComboBox control's FlatStyle is not displayed correctly in design and runtime when set FlatStyle property to Flat or Popup

### ISSUE THREAD

**Issue by John-Qiao** [dotnet/winforms#7727]
### .NET version

8.0.100-alpha.1.22430.2

### Did it work in .NET Framework?

Yes

### Did it work in any of the earlier releases of .NET Core or .NET 5+?

Yes, it works in .NET 6.0.
![6 0-Aquatic](https://user-images.githubusercontent.com/45864985/188091624-debb1e8b-60fe-4f12-bba5-0f2a4226e35e.png)

### Issue description

On Windows 11 and Windows 10 all high contrast themes, the ComboBox control's FlatStyle is not displayed correctly in design and runtime when set FlatStyle property to Flat or Popup.

Windows 11 HC Aquatic/Desert/Dusk/Night Sky testing results:
![Win11-1](https://user-images.githubusercontent.com/45864985/188093170-8303c05b-23d3-4d8e-b76f-8310914844b2.png)
![Win11-2](https://user-images.githubusercontent.com/45864985/188093213-746dfdff-87fd-4d7a-83ff-a7e48ec1fcea.png)
![Win11-3](https://user-images.githubusercontent.com/45864985/188093242-c2d4547a-a91d-4d82-ac46-b145c0fd4101.png)
![Win11-4](https://user-images.githubusercontent.com/45864985/188093280-188c0f7a-9585-4401-8472-590a3986388f.png)

Windows 11 default theme:
![Win11-Expected](https://user-images.githubusercontent.com/45864985/188093456-de5a9967-929c-4be5-af5c-2b68ea370705.png)

Windows 10 HC #1/#2/black/white testing result:
![Win10-1](https://user-images.githubusercontent.com/45864985/188093533-14c8fba7-9ff2-4c93-a9e5-26735d7d02f5.png)
![Win10-2](https://user-images.githubusercontent.com/45864985/188093559-f4fdb6fa-03b8-4351-8d42-3a87c8c020b0.png)
![Win10-3](https://user-images.githubusercontent.com/45864985/188093590-23619cae-107a-42dd-85ae-51d51d690ed9.png)
![Win10-4](https://user-images.githubusercontent.com/45864985/188093635-11acd434-d7e9-4f77-9e9e-4040f2a1eeb6.png)

Windows 10 default theme:
![Win10-Expected](https://user-images.githubusercontent.com/45864985/188093689-38587ee4-633e-4904-b82a-d533ddff3f37.png)

### Steps to reproduce

1. Set OS' theme to high contrast.
2. Open the [TestApp.zip](https://github.com/dotnet/winforms/files/9475963/TestApp.zip) project.
3. Open Form1 designer and observe the ComboBox controls in **Flat** and **Popup** columns.
4. Run the project, and observe the ComboBox controls in **Flat** and **Popup** columns too.

**Comment by merriemcgaw** [comment]
@Tanya-Solyanik - for this one I think that the complaint is that we can see the borders, whereas in Win 10/11 the default theme shows with no/white borders. This is OK behavior, otherwise how could someone tell that there is a control there. I like the new look for the new themes and propose no change here. Thoughts?

**Comment by merriemcgaw** [comment]
Confirmed with @Tanya-Solyanik that this is now by design and we like it this way.
